src/common/schemas/specifications/account-delete.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "accountDelete",
+  "link": "account-delete",
+  "type": "object",
+  "properties": {
+    "destination": {
```

Comment:
So are these properties mutually exclusive? 

Should we only be using X-Addresses?

Reply:
Great catch! I’m fine with only using X-addresses here. Will update the PR later today

Reply:
A product decision is to whether we want to support X-Addresses only. 

If you want both APIs, that's fine. But I think you then need a `oneof` in your `required` property below.

Reply:
Good point, it should be fine to support both classic addresses and X-addresses. Updated.